### DISCUSSION TITLE
Proxy often has error logs（no memory）

### DISCUSSION THREAD

**Discussion by J1a-wei** [Kong/kubernetes-ingress-controller#3338] [Q&A]
We often have error logs recently, suggesting that we have no memory, but we have observed that PROXY memory has not accounted for the host of the PROXY memory through Promethos. Is there any problem?
```
[error] 1097#0: *720540820 [lua] events.lua:273: post(): worker-events: failed posting event "healthy" by "lua-resty-healthcheck [0dc6f45b-8f8d-40d2-a504-473544ee190b:ethereum-mainnet-api.web3-helm.8599.svc]"; no memory

prometheus.lua:839: log_error(): Error while setting 'memory_workers_lua_vms_bytes{pid="1104",kong_subsystem="http"}' to '160569064': 'no memory'

log_error(): Error while setting 'upstream_target_health{upstream="heco-mainnet-api-graphql.web3-helm.8545.svc",target="172.1.19.101:8545",address="172.1.19.101:8545",state="dns_error",subsystem="http"}' to '0': 'no memory'

```

Kong Version: 2.7.0 (Helm chart)

**Comment by J1a-wei** [comment]
grafana snapshot
<img width="1508" alt="image" src="https://user-images.githubusercontent.com/49090950/210922406-84521f19-41d2-48df-8924-b58f99f8e09a.png">

<img width="1516" alt="image" src="https://user-images.githubusercontent.com/49090950/210950727-08f57162-1b1b-42b9-b455-e813a0021c22.png">


<img width="1513" alt="image" src="https://user-images.githubusercontent.com/49090950/210951239-33b0413e-5820-4b05-90ec-a595a0566c8d.png">
The third picture seems to be limited to 200MIB. Where can I configure this parameter?

**Comment by randmonkey** [comment]
I think you could configure the resource limits of containers and pods.
